src/natcap/invest/ui/urban_nature_access.py

```diff
@@ -0,0 +1,66 @@
+# coding=UTF-8
```

Comment:
Should we still add this over? I believe `ui/` has been pulled out already in the 3.13.0 release branch.

Reply:
Shoot, you're right.  I missed that.  I'll PR a thing back into the UNA branch.

Reply:
@dcdenu4 https://github.com/natcap/invest/pull/1248